src/Microsoft.Health.Fhir.Core/Features/Search/SearchValues/QuantitySearchValue.cs

```diff
@@ -89,7 +89,7 @@ public static QuantitySearchValue Parse(string s)
             }
 
             decimal quantity;
-            if (!decimal.TryParse(parts[0], NumberStyles.Number, CultureInfo.InvariantCulture, out quantity))
+            if (!decimal.TryParse(parts[0], NumberStyles.Number | NumberStyles.AllowExponent, CultureInfo.InvariantCulture, out quantity))
```

Comment:
https://www.hl7.org/fhir/STU3/datatypes.html#decimal
>decimals may not use exponents, and leading 0 digits are not allowed

Technically STU3 shouldn't support that notation, but in same time I'm not sure how strict we should be about that.
Any thoughts? @brendankowitz

Reply:
Good point! 
When I was testing my changes with Postman, I noticed that the POST on Observation is working also with the exponential format - I tried with both R4 and STU3. So if the final decision will be not to accept it in the GET for the STU3... well IMHO then probably it should not be accepted in the POST.

Reply:
It might be fine to allow this, since the community serializes already allow this. The behavior is additive rather than altering an expected outcome.
//cc @CaitlinV39